test/e2e/metadata_api_test.go

```diff
@@ -50,6 +50,7 @@ func TestMetadataAPI_Fanout(t *testing.T) {
 		[]string{sidecar1.GRPCNetworkEndpoint(), sidecar2.GRPCNetworkEndpoint()},
 		nil,
 		nil,
+		nil,
 		[]string{sidecar1.GRPCNetworkEndpoint(), sidecar2.GRPCNetworkEndpoint()},
```

Comment:
The order needs to be changed. In this test case, metadata endpoints should be `[]string{sidecar1.GRPCNetworkEndpoint(), sidecar2.GRPCNetworkEndpoint()}` and exemplar endpoints should be nil.